Add conflict-of-interest-policy.md

rstuart opened the pull request:
This policy was adopted on 2025-07-15 from the ACNC template found at:

    https://www.acnc.gov.au/tools/templates/conflict-interest-policy

Modifications made:

- Converted to markdown, and
- Template sections enclosed in [..] filled on.

rstuart committed:
Add conflict-of-interest-policy.md

This policy was adopted on 2025-07-15 from the ACNC template found at:

    https://www.acnc.gov.au/tools/templates/conflict-interest-policy

Modifications made:

- Converted to markdown, and
- Template sections enclosed in [..] filled on.

M0les commented on conflict-of-interest-policy.md:
```diff
@@ -0,0 +1,170 @@
+
+#### 5.3. Confidentiality of disclosures
+
+Only the committee members members will have access to the register
```
"members members"? Seems like a duplicated word typo.

jwoithe replied:
Yes, agreed.

rstuart replied:
Fixed.

M0les reviewed (dismissed):
Having given it a quick scan, I found what looks like a simple typo. Other than that, I didn't notice any semantic issues with the document.

jwoithe commented:
In general this looks reasonable, as would be expected given it's based on the ACNC template.  I second @M0les with the duplicated "members".
I noticed a few cases where it appears some punctuation has been lost, which makes certain paragraphs difficult to read. Comparison with the original template indicates that in all cases this has come about due to the loss of bullets in a bullet list. This makes all such items run together. Affected sections: 5, 5.1, 5.2, 6.1, 6.2. Suggested fixes follow.

jwoithe commented on conflict-of-interest-policy.md:
```diff
@@ -0,0 +1,170 @@
+identifying, disclosing and managing related party transactions and
+conflicts of interest across the charity monitoring compliance with
+this policy, and reviewing this policy on an annual basis to ensure
+that the policy is operating effectively.
```
Missing bullets.  Replace with the following.

```
The committee is responsible for:
  * establishing a system for identifying, disclosing and managing related
    party transactions and conflicts of interest across the charity
  * monitoring compliance with this policy, and
  * reviewing this policy on an annual basis to ensure that the policy is
    operating effectively.
```

rstuart replied:
Fixed.

jwoithe commented on conflict-of-interest-policy.md:
```diff
@@ -0,0 +1,170 @@
+identify and disclose any conflicts of interest carefully manage any
+conflicts of interest, and follow this policy and respond to any
+breaches.
+
```
Missing bullets, which are needed for clarity.  Replace with the following.

```
Linux Australia will manage conflicts of interest by requiring
committee members to:
  * avoid conflicts of interest where possible
  * identify and disclose any conflicts of interest carefully manage any conflicts
    of interest, and
  * follow this policy and respond to any breaches.
```

rstuart replied:
Fixed.

jwoithe commented on conflict-of-interest-policy.md:
```diff
@@ -0,0 +1,170 @@
+a resolution in a general meeting, consider appointing new committee
+members; and note that its committee members may consider whether
+they can continue in the role.
+
```
Missing bullets.  Replace with the following.

```
Linux Australia may:
  * obtain professional advice,
  * consider whether there are any relevant exceptions in its governing
    document or legislation,
  * consider whether it is appropriate for members to pass a resolution in a
    general meeting,
  * consider appointing new committee members; and
  * note that its committee members may consider whether they can continue
    in the role.
```

rstuart replied:
Fixed.

jwoithe commented on conflict-of-interest-policy.md:
```diff
@@ -0,0 +1,170 @@
+decide whether or not those conflicted committee members should:
+vote on the matter (this is a minimum), participate in any debate,
+or be present in the room during the debate and the voting.
+
```
Missing bullets.  Replace with the following.

```
Once the conflict of interest has been appropriately disclosed, the
committee (excluding the committee member who has made the disclosure, as
well as any other conflicted committee member) must decide whether or not
those conflicted committee members should:
  * vote on the matter (this is a minimum),
  * participate in any debate, or
  * be present in the room during the debate and the voting.

```

rstuart replied:
Fixed.

jwoithe commented on conflict-of-interest-policy.md:
```diff
@@ -0,0 +1,170 @@
+resources, and the possibility of creating an appearance of improper
+conduct that might impair confidence in, or the reputation of, the
+charity.
+
```
Missing bullets.  Replace with the following.

```
In deciding what approach to take, the committee will consider:
  * whether the conflict needs to be avoided or simply documented
  * whether the conflict will realistically impair the disclosing person’s
    capacity to impartially participate in decision-making
  * alternative options to avoid the conflict
  * the charity’s objects and resources, and
  * the possibility of creating an appearance of improper conduct that might
    impair confidence in, or the reputation of, the charity.
```

rstuart replied:
Looks like that's a direct copy from the URL, which I confess I hadn't read.  But now you've made me read it, it looks to have it's own issues(!).  I suspect it's meant to be:

In deciding what approach to take, the committee will consider:
  * whether the conflict needs to be avoided or simply documented,
  * whether the conflict will realistically impair the disclosing person’s
    capacity to impartially participate in decision-making,
  * alternative options to avoid the conflict with the charity’s objects and resources, and
  * the possibility of creating an appearance of improper conduct that might
    impair confidence in, or the reputation of, the charity.

rstuart replied:
Fixed.

rstuart committed:
Fix bullet formatting, grammar, and change "charity" to "organisation"

- Fix markdown formatting - thanks Miles and Jonathan.
- Fix repeat of "members" in 5.3 - thanks Miles.
- Change "charity" to "organisation" - thanks Kathy.

joeladdison committed:
Adjust confidentiality in conflict of interest policy

Change confidentiality clause to align with NSW Incorporated
Associations Act which requires disclosure of interests
register to be available to members upon request.

neillc commented on conflict-of-interest-policy.md:
```diff
@@ -0,0 +1,183 @@
+  is operating effectively.
+
+The organisation must ensure that its committee members are aware of
+the ACNC Governance Standards, particularly Governance Standard 5,
```
Should we link to the standard? https://www.acnc.gov.au/for-charities/manage-your-charity/governance-hub/5-duties-responsible-people

neillc commented on conflict-of-interest-policy.md:
```diff
@@ -0,0 +1,183 @@
+
+Once an actual, potential or perceived conflict of interest is
+identified, it must be entered into Linux Australia’s register of
+interests, as well as being raised with the committee.
```
Should we link to the register?

neillc commented on conflict-of-interest-policy.md:
```diff
@@ -0,0 +1,183 @@
+interest (including the nature and extent of the conflict of
+interest and any steps taken to address it).
+
+Linux Australia will also maintain a register of related party
```
Is this also a duty of the VP?

neillc commented on conflict-of-interest-policy.md:
```diff
@@ -0,0 +1,183 @@
+action against them.  This may include seeking to terminate their
+relationship with the organisation.
+
+If a person suspects that a committee member has failed to disclose
```
Any person, or a member of LA. I'm fine with any person, just want to confirm that is the intent.

hughhalf reviewed (commented):
All looks well reasoned to me, appreciate Russell and the rest of the committees efforts to put this in place.
